src/zemscripten.zig

```diff
@@ -21,6 +18,123 @@ pub const AnimationFrameCallback = *const fn (f64, ?*anyopaque) callconv(.C) c_i
 extern fn emscripten_request_animation_frame_loop(AnimationFrameCallback, ?*anyopaque) void;
 pub const requestAnimationFrameLoop = emscripten_request_animation_frame_loop;
 
+pub const EmscriptenResult = enum(i16) {
+    success = 0,
+    deferred = 1,
+    not_supported = -1,
+    failed_not_deferred = -2,
+    invalid_target = -3,
+    unknown_target = -4,
+    invalid_param = -5,
+    failed = -6,
+    no_data = -7,
+    timed_out = -8,
+};
+pub const CanvasSizeChangedCallback = *const fn (
+    i16,
+    *anyopaque,
+    ?*anyopaque,
+) callconv(.C) c_int;
+pub fn setResizeCallback(
+    cb: CanvasSizeChangedCallback,
+    use_capture: bool,
+    user_data: ?*anyopaque,
+) EmscriptenResult {
+    const result = emscripten_set_resize_callback_on_thread(
+        "2",
+        user_data,
+        @intFromBool(use_capture),
+        cb,
+        2,
+    );
+    return @enumFromInt(result);
+}
+extern fn emscripten_set_resize_callback_on_thread(
+    [*:0]const u8,
+    ?*anyopaque,
+    c_int,
+    CanvasSizeChangedCallback,
+    c_int,
+) c_int;
+
+pub fn getElementCssSize(
+    target_id: [:0]const u8,
+    width: *f64,
+    height: *f64,
+) EmscriptenResult {
+    return @enumFromInt(emscripten_get_element_css_size(
+        target_id,
+        width,
+        height,
+    ));
+}
+extern fn emscripten_get_element_css_size([*:0]const u8, *f64, *f64) c_int;
+
+/// EmmalocAllocator allocator
+/// use with linker flag -sMALLOC=emmalloc
+/// for details see docs: https://github.com/emscripten-core/emscripten/blob/main/system/lib/emmalloc.c
```

Comment:
These doc comments /// will be attributed only to the first subsequent declaration. We should use a standard // comment block here instead.